src/System.Diagnostics.Process/src/System/Diagnostics/Process.WinRT.cs

```diff
@@ -12,5 +12,49 @@ private bool StartCore(ProcessStartInfo startInfo)
                 ? throw new PlatformNotSupportedException(SR.UseShellExecuteNotSupported)
                 : StartWithCreateProcess(startInfo);
         }
+
+        private string GetMainWindowTitle()
+        {
+            throw new PlatformNotSupportedException();
+        }
+
+        public IntPtr MainWindowHandle
+        {
+            get
+            {
+                throw new PlatformNotSupportedException();
+            }
+        }
+
+        private bool CloseMainWindowCore()
+        {
+            throw new PlatformNotSupportedException();
+        }
+
+        public string MainWindowTitle
+        {
+            get
+            {
+                throw new PlatformNotSupportedException();
+            }
+        }
+
+        private bool IsRespondingCore()
+        {
+            throw new PlatformNotSupportedException();
```

Comment:
Can we please include a (localized) message in each of these - it can be the same one? Something folks can search the internet for.

Eg., "Operations on HWND-based windows are not supported on this platform."

Maybe @stephentoub has a better suggestion.

This also means that they don't leap out next time we grep or use a tool to find PNSE we should not have. These ones we should have, so they should have a message or at least a comment on the same line.